workbench/tests/invest/flaskapp.test.js

```diff
@@ -206,20 +168,45 @@ expect.extend({
   },
 });
 
-describe('Build each model UI from MODEL_SPEC', () => {
-  const { UI_SPEC } = require('../../src/renderer/ui_config');
-
-  test.each(Object.keys(UI_SPEC))('%s', async (model) => {
+describe('Test building model UIs and forum links', () => {
+
+  const models =  [

```

Comment:
I tried getting this list from the server, but couldn't use `test.each` on an array that had to be `await`ed. I then tried running them all in one test, but had issues with `render`ing multiple times. Simplest to just list them out here.